server/request.go

```diff
@@ -247,14 +308,14 @@ func (r *RpcRequest) sendTxToRelay() {
 
 	txTo := r.tx.To()
 	if txTo == nil {
-		r.writeRpcError("invalid target")
+		r.writeRpcError("invalid target", InternalError)
```

Comment:
It's a user error, so should probably be `InvalidRequest` rather than `InternalError`